manifests/init.pp

```diff
@@ -46,6 +46,8 @@
 #
 # $ipa_authentication::           Enable configuration for external authentication via IPA
 #
+# $ipa_authentication_api::       Enable configuration for external authentication via IPA for API
```

Comment:
I'm trying to imagine the workflows. When would you set `ipa_authentication` but set this to `false`? Wouldn't it more common that you keep the two in sync?

Or in other words: do we need this additional parameter?

Reply:
Maybe it's worth to rename the option to something more explicit, since this option is for enabling GSSAPI auth for API. Basic external auth will be enabled by default if `ipa_authentication` is `true`.